### PR TITLE
Design updates for Embedded Components

### LINKED ISSUE #9360: Design changes to the MOX
### Description

We need to make some tweaks to the MOX layout to align with the latest designs.
The main changes seem to be to the header bar, adding a divider, and using the Woo logo.

### Acceptance criteria

* Designs match the Figma

### Designs

**Figma:** xO8YdpKfaVoP2d8NvweZ5I-fi-1095_42589

### PR DESCRIPTION
**Pull request by dmallory42**
Fixes #9360

#### Changes proposed in this Pull Request

Note: we should hold off on merging this until the `dev/include-connect-js` PR is merged, and then target development.

This PR contains updates to the embedded onboarding flow to match the i4 designs: xO8YdpKfaVoP2d8NvweZ5I-fi-1095_43454

* Adds a border under the navbar.
* Updates the color and text of the button text on the nav bar to use theme colors.
* Update to the logo.
* Add left and right padding to add more separation
* Update the Stripe modal to use the Woo purple

#### Testing instructions

* Launch a new site with this branch using Jurassic Ninja ([quick link](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=dev/i4-design-updates))
* Go to the WCPay Dev Tools on the new JN site and enable the `Embedded KYC` checkbox, then save the change.
* Start a new onboarding. You should see the Embedded KYC as part of the MOX flow.
* Verify the page looks as expected, and the designs match the figma.

Screenshots:

Screenshot 2024-09-04 at 16 22 46

Screenshot 2024-09-04 at 16 22 54

Screenshot 2024-09-04 at 16 24 01

-------------------

- [ ] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by dmallory42** [comment]
cc. @orcunomattic @timmy5685 , would appreciate a quick check at the screenshots here to ensure they are in line with the expected desgin!

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `9381` or branch name `dev/i4-design-updates` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=dev/i4-design-updates) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 2a7be621c42d4b4c95b7b3742292301161f1d6e7
- Build time: 2024-09-19 09:43:40 UTC

_Note: the build is updated when a new commit is pushed to this PR._

**Comment by oaratovskyi** [comment]
@dmallory42 I'll review this one after design / product review JFYI

File: client/onboarding/style.scss [hunk 1]
```diff
@@ -27,10 +30,12 @@ body.wcpay-onboarding__body {
 					display: flex;
 					align-items: center;
 					padding: $gap-large;
+					color: #0073aaff;
```
Comment (mordeth) [inline client/onboarding/style.scss]: That's not a valid HEX color :)

File: client/onboarding/style.scss [hunk 2]
```diff
@@ -27,10 +30,12 @@ body.wcpay-onboarding__body {
...
 					font-size: 14px;
 
 					.gridicons-chevron-left {
 						margin-right: 2px;
+						fill: #0073aaff;
```
Comment (mordeth) [inline client/onboarding/style.scss]: Similar to the previous comment, this is not a valid HEX color.

**Review by mordeth — CHANGES_REQUESTED** [review]
Thank you for working on this Dan!

I’ve included some comments suggesting a few minor improvements.

Additionally, `Icon` and `closeSmall` are no longer used, perhaps we should consider deleting them?

File: client/onboarding/style.scss (hunk 1 above)
Comment (dmallory42) [reply client/onboarding/style.scss]: Thanks, I'm not sure where that color came from. I'm now using the color defined in the Figma.

File: client/onboarding/style.scss (hunk 2 above)
Comment (dmallory42) [reply client/onboarding/style.scss]: Same as above :)

**Comment by dmallory42** [comment]
Thank you for the review, @mordeth! I've addressed your suggestions and removed the unused imports. If you have time for another look, it's appreciated!

**Review by mordeth — APPROVED** [review]
It looks great after the changes. Excellent work!

**Review by oaratovskyi — APPROVED** [review]
@dmallory42 I've modified logo and the cancel button for the continue KYC page as well! Tried to refactor it, but it's tricky, so a bit of code duplication.

**Comment by timmy5685** [comment]
@dmallory42 - i don't know if this is related but when I create a store using the JN link above - the connect page is showing the sandbox mode notification at the top of the page before I do anything.

screencapture-devotedly-additional-squirrel-jurassic-ninja-wp-admin-admin-php-2024-09-06-16_18_35

Is that expected? The Stripe flow is also showing test mode - so maybe it's the way you set the JN to work?

Aside from that - when I try to run through the flow and complete the Stripe OTP step - I get a session expired message and can't proceed. When I click add information again the window pops for a second and then takes me right back here.

Screenshot 2024-09-06 at 4 25 35 PM

**Comment by mordeth** [comment]
>i don't know if this is related but when I create a store using the JN link above - the connect page is showing the sandbox mode notification at the top of the page before I do anything.

**Comment by mordeth** [comment]
>  i don't know if this is related but when I create a store using the JN link above - the connect page is showing the sandbox mode notification at the top of the page before I do anything.

> Is that expected? The Stripe flow is also showing test mode - so maybe it's the way you set the JN to work?

@timmy5685  The behavior you’re observing is expected due to the presence of the Dev Tools plugin, which is enabled by default on all JN sites created from the GitHub link and forces sandbox/dev mode. If you disable the Dev Tools plugin, both the sandbox mode and the notification should disappear.

**Comment by timmy5685** [comment]
ok - thanks for confirming @mordeth!

So that is also what is causing the expired session issue?

**Comment by dmallory42** [comment]
> So that is also what is causing the expired session issue?

No, I don't think so - I'm looking into this one 🙂  do you remember if you left the page open for a while when you saw that error, by any chance?

**Comment by dmallory42** [comment]
Just to confirm, we merged a PR last week that should address any session issues. Please let us know if you spot anything like that again.

**Comment by oaratovskyi** [comment]
> Just to confirm, we merged a PR last week that should address any session issues. Please let us know if you spot anything like that again.

cc @timmy5685

**Comment by oaratovskyi** [comment]
@orcunomattic could you please take a look and review it design-wise? 🙏

**Comment by orcungogus** [comment]
@oaratovskyi, the _MOX + Stripe Embedded Onboarding_ screens look great!

My only comment (as I mentioned on Slack) is on a color update request for button, text link and inline text link elements used in the MOX flow, as well as the header top header we use for both MOX and Stripe Embedded Onboarding:
Color
color-update1

The other color change request (if possible) I have is for the "Continue" CTA on the Stripe authentication pop-up: it was a blue button (in <Gutenberg Blueberry> color) previously, if we can revert it back to that WITHOUT affecting the top header area (which we switched to <Woo Purple 50>), that would be great, I am not sure if those two are connected:
Color 2
color-update2

**Comment by oaratovskyi** [comment]
@orcunomattic thanks for the review!
About the change requests, let me share what I've explored:

> color update request for button, text link and inline text link elements used in the MOX flow, as well as the header top header we use for both MOX and Stripe Embedded Onboarding

The thing is that currently the order of the colors are next: 
- `--wp-components-color-accent` not set
- `--wp-admin-theme-color` Gutenberg Blue
- `#3858E9` Gutenberg blueberry
Our Embedded Stripe KYC ignores the theme color and is always Gutenberg Blueberry. 
I've updated the MOX to ignore the theme color too (image 1). For the focus of CTA button I used Gutenberg Blueberry 10% darker (image 2). Does it look good to you? 
image
image

I thought about the consequence of this change: we have Gutenberg Blue CTA on the connect page and all other WooCommerce places and MOX + Stripe KYC having Gutenberg Blueberry. Could you confirm it's fine for now?
image

> The other color change request (if possible) I have is for the "Continue" CTA on the Stripe authentication pop-up: it was a blue button (in color) previously, if we can revert it back to that WITHOUT affecting the top header area (which we switched to <Woo Purple 50>), that would be great, I am not sure if those two are connected

Unfortunately, they are connected. `primaryColor` variable sets both.

**Comment by oaratovskyi** [comment]
@timmy5685 As I've implemented @orcunomattic suggestions - merging this one and preparing the final build for you to test 🚀  
p.s. will wrangle any tweaks if needed in the follow-up PR

**Comment by timmy5685** [comment]
@orcunomattic for the CTA focus - I think it should be dark blueberry `#1D35B4`, right? Checking the style guide and I think that's what I'm seeing.

**Comment by oaratovskyi** [comment]
> for the CTA focus - I think it should be dark blueberry #1D35B4, right? Checking the style guide and I think that's what I'm seeing.

@timmy5685 @orcunomattic I wrangled a PR to address this https://github.com/Automattic/woocommerce-payments/pull/9481

**Pull request merged**